### DISCUSSION TITLE
任务4 是否一定需要使用ethers.js

### DISCUSSION THREAD

**Discussion by zhangolve** [openbuildxyz/Web3-Frontend-Bootcamp#790] [Q&A]
任务4的说明里提到了需要使用ethers.js ,但是似乎wagmi v2天然对viem支持更友好，是否可以使用viem而不使用ethers.js 

FYI:

https://github.com/openbuildxyz/Web3-Frontend-Bootcamp/blob/main/task/04_NFTMarket_Component.md
https://wagmi.sh/react/guides/viem

**Comment by zhengqiuwan** [comment]
可以的
